Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -71,6 +71,7 @@ public static IMvcBuilder AddJellyfinApi(this IServiceCollection serviceCollecti
                 // Clear app parts to avoid other assemblies being picked up
                 .ConfigureApplicationPartManager(a => a.ApplicationParts.Clear())
                 .AddApplicationPart(typeof(StartupController).Assembly)
+                .AddApplicationPart(typeof(NotificationsController).Assembly)
```

Comment:
The is the same assembly as the one above, why add it?

Reply:
Honestly, it was 2AM is my only justification 😅. It's been removed in the latest commit.